Backend/app/rider-platform/rider-app/Main/spec/Storage/Booking.yaml

```diff
@@ -29,6 +29,7 @@ imports:
   VehicleServiceTier: Domain.Types.VehicleServiceTier
   VehicleServiceTierType: Domain.Types.VehicleServiceTier
   HighPrecMoney: Kernel.Types.Common
+  Estimate: Domain.Types.Estimate
```

Comment:
Not required to add `estimateId` as it is same as `itemId`
we can rename domain type to be `bppEstimateId` for `itemId`